bobtemplates/hooks.py

```diff
@@ -141,6 +143,14 @@ def pre_dexterity_type_name(configurator, question):
         raise SkipQuestion
 
 
+def post_dexterity_type_name(configurator, question, answer):
```

Comment:
@pbauer wouldn't it make sense to write a few unit tests to show how this is supposed to work and to make sure the function works as expected?